core/src/main/scala/org/typelevel/otel4s/otel4s.scala

```diff
@@ -130,19 +158,81 @@ object Meter {
 trait SyncInstrumentBuilder[F[_], A] {
   type Self <: SyncInstrumentBuilder[F, A]
 
+  /** Sets the unit of measure for this instrument.
+    *
+    * @see
+    *   <a
+    *   href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument-unit">Instrument
+    *   Unit</a>
+    *
+    * @param unit
+    *   the measurement unit. Must be 63 or fewer ASCII characters.
```

Comment:
I don't know what happens when it's not.  Probably an exception?

Reply:
OpenTelemetry uses the empty string `""` instead.